sw/host/opentitanlib/src/spiflash/flash.rs

```diff
@@ -150,13 +123,14 @@ impl SpiFlash {
         let mut buf = vec![0u8; 256];
         let mut tries = 0;
         loop {
-            spi.run_transaction(&mut [
-                // READ_SFDP always takes a 3-byte address followed by a dummy
-                // byte regardless of address mode.  To simplify, we always
-                // read from address zero.
-                Transfer::Write(&[SpiFlash::READ_SFDP, 0, 0, 0, 0]),
-                Transfer::Read(&mut buf),
-            ])?;
+            // READ_SFDP always takes a 3-byte address followed by a dummy byte regardless of
+            // address mode.  To simplify, we always read from address zero, and specifying 4-byte
+            // addressing mode achieves sending of the dummy byte.
```

Comment:
This should probably be turned into some sort of FIXME or TODO so that eventually, READ_SFDP gets represented properly as `MODE_111.dummy_cycles(8).cmd_addr(...)`.

Reply:
Of course